samples/sampler/package.json

```diff
@@ -24,8 +24,8 @@
   },
   "dependencies": {
     "@enact/agate": "../../",
-    "@enact/core": "^3.2.5",
-    "@enact/ui": "^3.2.5",
+    "@enact/core": "^3.3.0-alpha.8",
```

Comment:
revert this change as it is part of PR FORTECH-7

Reply:
done